dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/SymbolProblem.java

```diff
@@ -85,12 +84,25 @@ public int hashCode() {
   }
 
   @Override
-  public String toString() {
-    return MoreObjects.toStringHelper(this)
-        .omitNullValues()
-        .add("errorType", errorType)
-        .add("symbol", symbol)
-        .add("containingClass", containingClass)
-        .toString();
+  public final String toString() {
+    StringBuilder builder = new StringBuilder(symbol.toString());
+    switch (getErrorType()) {
```

Comment:
Switches on type usually signal that the object hierarchy isn't being fully used. Here the relevant info can be pushed into ErrorType, and this becomes something like `getErrorType().getMessage()` with no switch.